### PR TITLE
[Notifier] Add Ntfy bridge

### PR DESCRIPTION
**Pull request by mikaelkael**
In addition with https://github.com/symfony/symfony/pull/50131 for the new Ntfy notifier bridge

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -420,15 +420,16 @@ The push channel is used to send notifications to users by using
...
-PagerDuty       ``symfony/pager-duty-notifier``        ``pagerduty://TOKEN@SUBDOMAIN``
-Pushover        ``symfony/pushover-notifier``          ``pushover://USER_KEY:APP_TOKEN@default``
-==============  ====================================  =================================================================================
+=================  ====================================  =================================================================================
+Service            Package                               DSN
+=================  ====================================  =================================================================================
+`Engagespot`_      ``symfony/engagespot-notifier``       ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
+`Expo`_            ``symfony/expo-notifier``             ``expo://Token@default``
+`OneSignal`_       ``symfony/one-signal-notifier``       ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID``
+`Ntfy`_            ``symfony/ntfy-notifier``             ``ntfy://default/topic_name``
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
`Ntfy`_            ``symfony/ntfy-notifier``             ``ntfy://default/TOPIC``
```

File: notifier.rst [hunk 2]
```diff
@@ -420,15 +420,16 @@ The push channel is used to send notifications to users by using
...
+`PagerDuty`_       ``symfony/pager-duty-notifier``       ``pagerduty://TOKEN@SUBDOMAIN``
+`Pushover`_        ``symfony/pushover-notifier``         ``pushover://USER_KEY:APP_TOKEN@default``
+=================  ====================================  =================================================================================
 
 .. versionadded:: 6.1
```
Comment (OskarStark) [inline notifier.rst]: Please add your bridge to the versionadded 6.3 block in line 440

**Comment by OskarStark** [comment]
Would you open a PR against 5.4 branch for `and links to push bridges' README`?

That would be helpful

File: notifier.rst (hunk 2 above)
Comment (mikaelkael) [reply notifier.rst]: Oh :open_mouth:, didn't see that before, sorry!

**Comment by mikaelkael** [comment]
> Would you open a PR against 5.4 branch for `and links to push bridges' README`?
> 
> That would be helpful

Only against 5.4 branch? Not 6.x other versions ?

**Comment by mikaelkael** [comment]
@OskarStark : I will review this PR after merge of https://github.com/symfony/symfony-docs/pull/18257 to only have addition of Ntfy bridge (not all bridges links)

File: notifier.rst (hunk 2 above)
Comment (OskarStark) [reply notifier.rst]: Don't worry 😃

**Comment by OskarStark** [comment]
Rebase unlocked ✅ 

Thank you for the other Pr's 🙏

File: notifier.rst [hunk 3]
```diff
@@ -420,23 +420,24 @@ The push channel is used to send notifications to users by using
...
+=================  ====================================  =================================================================================
+Service            Package                               DSN
+=================  ====================================  =================================================================================
+`Engagespot`_      ``symfony/engagespot-notifier``       ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
+`Expo`_            ``symfony/expo-notifier``             ``expo://Token@default``
+`OneSignal`_       ``symfony/one-signal-notifier``       ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID``
+`Ntfy`_            ``symfony/ntfy-notifier``             ``ntfy://default/TOPIC``
+`PagerDuty`_       ``symfony/pager-duty-notifier``       ``pagerduty://TOKEN@SUBDOMAIN``
+`Pushover`_        ``symfony/pushover-notifier``         ``pushover://USER_KEY:APP_TOKEN@default``
+=================  ====================================  =================================================================================
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
===============  ====================================  =================================================================================
Service          Package                               DSN
===============  ====================================  =================================================================================
`Engagespot`_    ``symfony/engagespot-notifier``       ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
`Expo`_          ``symfony/expo-notifier``             ``expo://Token@default``
`OneSignal`_     ``symfony/one-signal-notifier``       ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID``
`Ntfy`_          ``symfony/ntfy-notifier``             ``ntfy://default/TOPIC``
`PagerDuty`_     ``symfony/pager-duty-notifier``       ``pagerduty://TOKEN@SUBDOMAIN``
`Pushover`_      ``symfony/pushover-notifier``         ``pushover://USER_KEY:APP_TOKEN@default``
===============  ====================================  =================================================================================
```

Comment (OskarStark) [reply notifier.rst]: Lets keep the smaller table like before please

**Review by OskarStark — APPROVED** [review]
After my comment

**Review by OskarStark — APPROVED** [review]

File: notifier.rst [hunk 4]
```diff
@@ -436,7 +437,7 @@ Service          Package                               DSN
 
 .. versionadded:: 6.3
 
-    The PagerDuty and Pushover integrations were introduced in Symfony 6.3.
+    The Ntfy, PagerDuty and Pushover integrations were introduced in Symfony 6.3.
```
Comment (OskarStark) [inline notifier.rst]: Please add a 6.4 block, as we are in feature freeze phase right now.

**Review by OskarStark — APPROVED** [review]

File: notifier.rst (hunk 4 above)
Comment (mikaelkael) [reply notifier.rst]: @OskarStark : yes, I will review it when the 6.4 branch will be ready

Comment (OskarStark) [reply notifier.rst]: It is ready now

File: notifier.rst [hunk 5]
```diff
@@ -426,6 +426,7 @@ Service          Package                               DSN
 `Engagespot`_    ``symfony/engagespot-notifier``       ``engagespot://API_KEY@default?campaign_name=CAMPAIGN_NAME``
 `Expo`_          ``symfony/expo-notifier``             ``expo://Token@default``
 `OneSignal`_     ``symfony/one-signal-notifier``       ``onesignal://APP_ID:API_KEY@default?defaultRecipientId=DEFAULT_RECIPIENT_ID``
+`Ntfy`_          ``symfony/ntfy-notifier``             ``ntfy://default/TOPIC``
```
Comment (OskarStark) [inline notifier.rst:434]: Please move above OneSignal, thanks

Comment (mikaelkael) [reply notifier.rst:434]: I just rebased, I will review and add a 6.4 paragraph 😉

**Comment by mikaelkael** [comment]
@OskarStark : doc PR against 6.4, code rebased for 6.4 and recipes added :blush:

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Merged! Thanks Mickael.

**Pull request merged**